### ISSUE TITLE
EFR32BGx Bluetooth Support

### ISSUE THREAD

**Issue by ravnicas** [zephyrproject-rtos/zephyr#28938]
Hello folks,

im using an EFR32BG1P part which is at least at the moment not in zephyrs device tree, but the EFR32BG13P is, which is almost the same part.
I've added the missing definitions for my part and board efr32_radio_brd4100a which would be similar to the existing efr32_radio_brd4104a board. I can compile and run simple samples, but none of the bluetooth samples do start or to some extent even compile.

The error im getting is:
```
[00:00:00.000,000] <inf> flash_gecko: Device FLASH_CTRL initialized
[00:00:00.027,000] <err> bt_hci_core: No HCI driver registered
```

or:
```
Bluetooth init failed (err -19)
```

since the Kconfig.defconfig file with efr32_radio looks like this:
```
# EFR32 radio board configuration``
# Copyright (c) 2020 Piotr Mienkowski
# SPDX-License-Identifier: Apache-2.0

config BOARD_EFR32_RADIO
	bool

config BOARD_INIT_PRIORITY
	int
	default KERNEL_INIT_PRIORITY_DEFAULT
	depends on BOARD_EFR32_RADIO
	help
	  Board initialization priority. This must be bigger than
	  GPIO_GECKO_COMMON_INIT_PRIORITY.
```

Im wondering if the bluetooth support is at least not configured properly. After digging around i've found some definitions of:
```
config BT_CTRL
    default BT
```
within nrf52 boards.

Also no mention of efr32 boards with ble support within the documentation.

**Comment by carlescufi** [comment]
Is this a duplicate of https://github.com/zephyrproject-rtos/zephyr/issues/11773?

This is blocked by https://github.com/zephyrproject-rtos/zephyr/issues/17005
